tests/fixtures/addon/npm/package.json

```diff
@@ -14,11 +14,13 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
```

Comment:
`test:ember-compat` to make it easier to type?

Reply:
I don't know that folks will type it much? I'd prefer to avoid the abbreviation